alot/commands/globals.py

```diff
@@ -414,9 +414,24 @@ def __init__(self, buffer=None, force=False, redraw=True, **kwargs):
 
     @inlineCallbacks
     def apply(self, ui):
+        def last_buffer_no_close():
+            """Helper that prints a message to the log and notifies if this is
+            the last buffer and the settings don't allow closebuffer to
+            exit."""
+            msg = ('not closing last remaining buffer as '
+                   'global.quit_on_last_bclose is set to False')
+            logging.info(msg)
+            ui.notify(msg, priority='error')
+
         if self.buffer is None:
             self.buffer = ui.current_buffer
 
+        # If there is only one buffer and the settings don't allow using
+        # closebuffer to exit, then just stop.
```

Comment:
Is it possible to close the last buffer without closing? I tested closing the last buffer without exiting, and I'm getting a stack trace because the code assumes that there will always be one buffer open. The current behavior is to not close the last buffer, but without a notification of why.

Maybe it would be better to have a comment like:
Alot always needs to have at least one buffer opened, if this is the last buffer and the configuration is not quit when the last buffer closes then don't close the buffer, and provide a message as to why.